ECtools/statsd.cpp

```diff
@@ -205,7 +205,7 @@ static HRESULT sd_listen(ECConfig *cfg, struct socks &sk)
 	memset(&pfd, 0, sizeof(pfd));
 	pfd.events = POLLIN;
 	for (const auto &spec : tokenize(cfg->GetSetting("statsd_listen"), ' ', true)) {
-		auto ret = ec_listen_generic(spec.c_str(), &pfd.fd);
+		auto ret = ec_listen_generic(spec.c_str(), &pfd.fd, S_IRWUG | S_IROTH | S_IWOTH, cfg->GetSetting("run_as_user"), cfg->GetSetting("run_as_group"));
```

Comment:
Can you elaborate on your choice to include S_IxOTH for statsd.cpp but leave it out for idx_net.cpp?

Reply:
Initial idea was to give the permissions a config-option and start with relaxed settings as default.
That's the approach I'm using  on our private build, as of current. 

Concerning this PR, I decided to set defaults as they were created on our CentOS6-system prior the recent refactorings  - so this is not an analytical decision.

Likely, there's no sense and need to expose statsd and indexd  to IxOTH - imho, group and user generally offer sufficent possibilites to both control and document security relations, thus  S_IxOTH  should be dropped for all sockets. But eventually this could break the universe...